### ISSUE TITLE
Unable to allow accessibility on pixel 9

### ISSUE THREAD

**Issue by Gairbair129** [jonasbark/swiftcontrol#53]
Downloaded app but stuck trying to allow accessibility to continue with permissions. Any tricks? Google's answer to open app and allow restricted access is not available.

**Comment by Gairbair129** [comment]
I found out the three dots open the accessibility option in the app so you can disregard my issue.🙂‍↕️
